packages/esm-patient-lists-app/src/action-button/patient-lists-action-button.extension.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Events } from '@carbon/react/icons';
+import { SiderailNavButton, launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+function PatientListsActionButton() {
+  const { t } = useTranslation();
+
+  const handleLaunchPatientListsWorkspace = useCallback(() => launchPatientWorkspace('patient-lists'), []);
```

Comment:
This could be hoisted out of the component altogether, e.g.,

```ts
const handleLaunchPatientListsWorkspace = () => launchPatientWorkspace('patient-lists');

function PatientListsActionButton() 
```